src/main/java/org/aion/wallet/account/AccountManager.java

```diff
@@ -320,13 +299,28 @@ private String getStoredAccountName(final String publicAddress) {
         return walletStorage.getAccountName(publicAddress);
     }
 
+    private AccountDTO getNewAccount(final String publicAddress, boolean isImported, int derivation) {
```

Comment:
`final` ftw

Reply:
method is private so final would be pointless

Reply:
I meant for the parameters :))